src/solve.jl

```diff
@@ -24,6 +24,30 @@ function solve(prob::BVProblem, alg::Shooting; kwargs...)
     sol
 end
 
+function solve(prob::BVProblem, alg::ParameterShooting; kwargs...)
+    bc = prob.bc
+    u0 = deepcopy(prob.u0)
+    # Form a root finding function.
+    loss = function (resid, minimizer)
```

Comment:
The minimizer is the parameters, not the initial condition, so it needs to be placed correctly.